holoviews/plotting/widgets/widgets.js

```diff
@@ -522,6 +522,7 @@ function init_dropdown(id, plot_id, dim, vals, value, next_vals, labels, next_di
 
 if (window.HoloViews === undefined) {
   window.HoloViews = {}
+  window.PyViz = {}
```

Comment:
Shouldn't ``window.HoloViews`` and ``window.PyViz`` be referencing the same object, rather than two distinct objects?

Reply:
The jlab extension for example still references ``windows.HoloViews`` so if they are not the same object things will not work.

Reply:
Good point.